### PR TITLE
added semantic description to social.media page

### PR DESCRIPTION
**Pull request by charlieintech**
## Type of PR

- [ ] ✨ Feature
- [ ] 🐞 Bug fix
- [ X] 📄 Documentation update
- [ ] Others

### CODE REVIEW

**Review by mariahlaqua — CHANGES_REQUESTED** [review]
Hi @charlieintech ! Thanks for your contribution. Can you update the description:

- remove // from beginning
- mention SheSharp (e.g. "A place for SheSharp community members to share their social media links")
- maybe include a link to join our Slack (https://shesharp.co/slack) in the description, or a section for SheSharp socials with the website and Slack.

Let me know if you need further input. Thanks so much for contributing to our project!

**Comment by mariahlaqua** [comment]
Also hi @charlieintech ! I did #100Devs cohort two! nice to see another 100Devs-er here!!

**Comment by charlieintech** [comment]
> Hi @charlieintech ! Thanks for your contribution. Can you update the description:
> 
> * remove // from beginning
> * mention SheSharp (e.g. "A place for SheSharp community members to share their social media links")
> * maybe include a link to join our Slack (https://shesharp.co/slack) in the description, or a section for SheSharp socials with the website and Slack (this is an extra bonus and not required if you would just like it approved )
> 
> Let me know if you need further input. Thanks so much for contributing to our project!

Hi! I made changes and pushed the changes. Please let me know if I need to make any corrections.

File: socials/socials.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+
+A place for SheSharp community members to share their social media links
+
+### Slack
+
+- https://shesharp.co/slack
+
+### SheSharp Site 
+
+- [SheSharp] (https://www.shesharp.co/new-in-tech-scholarship)
```
Comment (adiati98) [inline socials/socials.md:15]:
```suggestion
### Join SheSharp Community

Join our community on [Slack](https://shesharp.co/slack)!

For more information about our community, visit [our website](https://www.shesharp.co/).
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
@charlieintech I left here some feedback 🙂

**Comment by charlieintech** [comment]
Thanks for the help :) I think I updated it. Please let me know if there is anything else.

**Review by mariahlaqua — CHANGES_REQUESTED** [review]
`
Come and join out community on [Slack] (https://shesharp.co/slack)`

Can we fix this typo as well? Thanks again @charlieintech !!

`
Come and join our community on [Slack] (https://shesharp.co/slack)`

**Comment by charlieintech** [comment]
Fixed typo :)

**Comment by mariahlaqua** [comment]
Thank you @charlieintech ! 💯

**Pull request merged**